scripts/lib/install-node

```diff
@@ -9,6 +9,21 @@ fi
 
 source "$NVM_DIR/nvm.sh"
 node_version=6.6.0
+npm_version=3.10.3
+# if the hashed_value is not changed then exit quickly
+script_hash=`cat scripts/lib/install-node scripts/setup/node-wrapper scripts/setup/npm-wrapper`
```

Comment:
I've would have preferred to sha1sum each file separately instead of having a variable which contains the contents of the three files combined, however that could be just personal preference.